microsoft/terminal
# Share PCH files between WinRT components to speed up builds

## Summary of the Pull Request

Moves CppWinRT projects to use a shared PCH to speed up build time since they all require pretty much the same header things.


## PR Checklist
* [x] Closes #946


## Validation Steps Performed

Built it with msbuild, built it with VS2019. Now we'll try building with CI and hopefully confirm it's OK.
